File: DataSummitSolution/DataSummitUI/src/app/properties/models/drawingProperty.ts
export class DocumentProperty {

    Id: string;
    StandardName: string;
    Name: string;
    Value: string;
    Confidence: number;

    constructor(Id?: string,
        standardName?: string,
        name?: string, 
        value?: string,
        confidence?: number)
    {
        this.Id = Id || null; 
        this.StandardName = standardName || null; 
        this.Name = name || null; 
        this.Value = value || null; 
        this.Confidence = confidence || null; 
    }    
}

File: DataSummitSolution/DataSummitWeb/Models/DrawingProperty.cs
using DataSummitHelper.Dto;
using System;

namespace DataSummitWeb.Models
{
    public class DocumentProperty
    {
        public long ProfileAttributeId { get; set; }
        public string StandardName { get; set; }
        public string Name { get; set; }
        public decimal? Confidence { get; set; }
        public Guid SentenceId { get; set; }
        public string WordValue { get; set; }

        public static DocumentProperty FromDto(DocumentPropertyDto dto)
        {
            if (dto == null)
            {
                return null;
            }

            return new DocumentProperty
            {
                ProfileAttributeId = dto.ProfileAttributeId,
                StandardName = dto.StandardName,
                Name = dto.Name,
                Confidence = null,
                SentenceId = dto.SentenceId,
                WordValue = dto.WordValue
        };
        }
    }
}
